operator/internal/manifests/gateway_tenants_test.go

```diff
@@ -184,6 +185,8 @@ func TestConfigureDeploymentForMode(t *testing.T) {
 		dpl   *appsv1.Deployment
 		want  *appsv1.Deployment
 	}
+	name := "test"
+	ns := "test-ns"
```

Comment:
Although I understand that this aims to be a reusability win, I suggest we keep test cases below as independent as possible. Test cases should be isolated from each other even if that means that we have some duplication of inputs. That makes them side-effect free usually and in extend able to execute in parallel.

Reply:
makes sense